Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -2410,49 +2475,48 @@ private struct DateRange : Sequence, IteratorProtocol {
 extension Date {
     //
     // Returns the amount of years from another date
-    func years(from date: Date) -> Int {
+    private func years(from date: Date) -> Int {
```

Comment:
Rather than marking each individual method as `private`/`fileprivate`, you could mark the entire extension as private:

```swift
private extension Date { … }
```

Reply:
Marked extension `fileprivate`